[DOCS] Documents BulkHelper JSON ingestion (#2099)
elastic/elasticsearch-ruby: docs/helpers.asciidoc

helper.update(documents)
----

[discrete]
==== Ingest a JSON file

> `BulkHelper` also provides a helper to ingest data straight from a JSON file. You can pass in a file path, and th helper will parse and ingest the documents in the file:

Reviewer: ```suggestion
`BulkHelper` also provides a helper to ingest data straight from a JSON file. By giving a file path as an input, the helper will parse and ingest the documents in the file:
```

[source,ruby]
----
file_path = './data.json'
helper.ingest_json(file_path)
----

> In cases where the array of data you want to ingest is not necessarily in the root of the JSON file, you can pass in the keys to access the data, for example given the following JSON file:

Reviewer: ```suggestion
In cases where the array of data you want to ingest is not necessarily in the root of the JSON file, you can provide the keys to access the data, for example given the following JSON file:
```

[source,json]
----
{
  "field": "value",
  "status": 200,
  "data": {
    "items": [
      {
        "name": "Item 1",
        (...)
      },
      {
      (...)
    ]
  }
}
----

> You would write the following Ruby code to ingest it:

Reviewer: ```suggestion
The following is an example of the Ruby code to ingest the documents in the JSON above:
```
